docker/launchTomcat.sh

```diff
@@ -88,6 +88,9 @@ export CATALINA_OPTS="\
   -Dvisualize.fits.search.path=${VIS_PATH} \
 	"
 
+#----- override ADMIN_PROTECTED path if set
+if [ ! -z ${ADMIN_PROTECTED+x} ]; then export CATALINA_OPTS="${CATALINA_OPTS} -DADMIN_PROTECTED='${ADMIN_PROTECTED}'"; fi
```

Comment:
Are we really want to allow the admin path to be changed. I don't know if there is a use case for this.
We could do something like the following: defined a USE_ADMIN (which default to true) if false set the path to a empty string. 
```bash

export CATALINA_OPTS="${CATALINA_OPTS} -DADMIN_PROTECTED=$([ "$USE_ADMIN" == "FALSE" ] && echo '' || echo '/admin/*')
```

This seem less confusing in the config.

Reply:
I basically allow the `url-pattern` value of the ADMIN protected location to be overridden by an env.
It currently defaults to `/admin/*`.   Firefly have 2 endpoints under admin, `/admin/status` and `/admin/db`.
In practice, you can set ADMIN_PROTECTED=/admin/db, to still use basic auth on that endpoint.

But, if you rather create a single-use `{USE_ADMIN_AUTH:-true}` to set all or nothing, I am fine it with.

Reply:
I think it will be better to just have a boolean. I think it will be easier to support.

Reply:
done.

Reply:
That change is fine with me, and I think the RSP team would be more comfortable just turning something on or off rather than attempting to make distinctions between the two.  This way you can add whatever you want under `/admin` in the future and it will all just work.